scripts/format-code.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
I want to get it working; I can refine once it builds, runs and passes CI again.